src/apiCall.ts

```diff
@@ -1,6 +1,17 @@
+interface Data {
+  data: {
+    item: {
+      season: string; 
+      color: string;
+      clothing_type: string;
+      size: string;
+      notes: string;
+    }
+  } 
+}
+
 export const getAllItems = async () => {
   const url = "https://closet-manager-be.herokuapp.com/api/v1/users/1/items";
```

Comment:
Nothing needing fixing, just want to mention; Look into using environment variables (like a .env file) so that you do not have to hardcode URL's into your code. Allows an easy way to change URL's should patterns change, but also allows you to set a different variable per environment (local, vs staging, vs prod, etc)